curriculum/challenges/english/25-front-end-development/lab-favorite-icon-toggler/66bf6bacf178eac7b96d4f5e.md

```diff
@@ -45,55 +47,62 @@ Your individual list items should contain the item name.
 
 ```js
 assert.exists(document.querySelector('ul li').textContent);
-
 ```
 
-Your individual list item should contain a  `span` element with the class `favorite-icon` 
+Your individual list item should contain a `button` element with the class `favorite-icon`.
 
 ```js
-assert.exists(document.querySelector('ul li span.favorite-icon'));
+assert.exists(document.querySelector('ul li button.favorite-icon'));
 ```
 
-Initially, the `span` elements should contain the code `&#9825;` to represent an empty heart.
+Initially, the `button` elements should contain the code `&#9825;` to represent an empty heart.
 
 ```js
-const inputs = document.querySelectorAll('ul li span.favorite-icon');
-assert.isNotEmpty(inputs)
+const inputs = document.querySelectorAll('ul li button.favorite-icon');
+assert.isNotEmpty(inputs);
 
 for (let input of inputs) {
-    assert.strictEqual(input.innerHTML.charCodeAt(0), 9825);
+  assert.strictEqual(input.innerHTML.charCodeAt(0), 9825);
 }
 ```
 
-When the `span` element is clicked, and it contains the class `filled`, you should remove the class `filled` from the `span` element and change the innerHTML of the `span` element to `&#9825;`.
+The `filled` class should contain style information. 
```

Comment:
It's not mentioned in the user stories to specifically add styles to the `filled` class.
We should add a user story for the required style.